### DISCUSSION TITLE
[Projects Beta] Ability to sort issues independently in different views?

### DISCUSSION THREAD

**Discussion by roll** [community/community#15178] [Projects and Issues]
# Overview

Currently, when we change the order of issues in a view A it also changes all other views on the board. I guess it's intended behavior for many workflows but, for example, for ours sorting "Current" view should not mess with the order in "Backlog" etc.

Not sure how to friend both approaches. Maybe having an option for a view to be independently sorted...

**Comment by kfollesdal** [comment]
I am really liking the way ProjectsV2 (Project Beta) is going :+1:  It would be a very nice feature to be allowed to manual order/prioritize issues in views without changing manual order on all views that have manual sorting. 

Let me clarify. If you for example have to views in you project. Then you can have different sorting on each view, for example on finish date in one view and labels in another. But if you choose no storting (manual), you can drag issues around and make a manual sorting/prioritizing. "Problem" is that if you choose no sorting (manual) on booth views, you change the ordering in booth views, when you change order in one. 

It would be nice if manual ordering was dependent on view or have a option to lock ordering on view when manual ordering.

**Comment by kfollesdal** [comment]
This discussion should have a **project** label also :smile:

**Comment by nieznanysprawiciel** [comment]
I would say this is not only nice to have feature, but rather an unexpected behavior in some cases.
If you have "board" view, it is enough to move issues to different column to unconsciously change order in "table" views.
This means that you completely can't rely on priorities based on ordering, what is rather cumbersome.

**Comment by mikespokefire** [comment]
We also have this problem, and it is detrimental to our workflow, which is a shame because aside from this Projects has been working brilliantly for us.

We have 2 views in our Projects Beta

1. A "Planning" board in table layout, manually sorted
2. A "Board" view in board layout, sorting doesn't matter here, columned by a "Status" column (Backlog, Doing, Done)

The "Planning" board is manually sorted, so issues can be dragged and put into a priority order.

The "Board" view I've tried a few different sort orders (sorting doesn't matter) but when the "Status" of an issue changes, then this will cause the issue to move on the "Planning" board to move a seemingly random location on the list, losing where it was manually positioned to. This means we cannot trust our prioritised list of issues on the "Planning" board any more.

**Comment by jfritz** [comment]
I like the current behavior. Since priority is based on order, the same priority is reflected throughout all views of the same data. If you have different priorities you might consider a different project altogether.
